app/src/main/java/com/r5k/contacerveja/ui/drink/view/DrinkFragment.kt

```diff
@@ -164,4 +168,36 @@ class DrinkFragment : BaseFragment(),DrinkMVPView, View.OnClickListener {
         var mainActivity = activity as MainActivity
         mainActivity.removeDrinkFragment(drink)
     }
+
+    private fun showChangeAmountDialog(drink : Drink){
+        val builder = AlertDialog.Builder(context!!)
+
+        val view = layoutInflater.inflate(R.layout.dialog_change_amount, null)
+
+        val amountEditText = view.findViewById(R.id.edt_change_amount) as TextInputEditText
+
+        builder.setView(view)
+
+        builder.setPositiveButton(android.R.string.ok) { dialog, p1 ->
+            val newAmount = amountEditText.text
+            var isValid = true
+            if (newAmount!!.isBlank()) {
+                amountEditText.error = getString(R.string.validation_empty)
+                isValid = false
+            }
+
+            if (isValid) {
+                Log.d(TAG, "Changing drink amount to $newAmount")
+                presenter.changeDrinkAmount(drink, newAmount.toString().toInt())
+                dialog.dismiss()
+            }
+        }
+
+        builder.setNegativeButton(android.R.string.cancel) { dialog, p1 ->
+            dialog.cancel()
+        }
+
+        builder.show()
+    }
```

Comment:
This method should be in a DialogHelper

Reply:
There are a lot of duplicated code about dialog at here and in MainActivity, let's create this helper in another branch.

Reply:
I have created this issue #29